nikola/data/themes/base/templates/embedded-fotorama.tmpl

```diff
@@ -0,0 +1,16 @@
+## -*- coding: utf-8 -*-
+
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script> <!-- 33 KB -->
+
+<link  href="http://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.css" rel="stylesheet"> <!-- 3 KB -->
+<script src="http://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.js"></script> <!-- 16 KB -->
+
+<div class="fotorama"
+     % for key, value in options.iteritems():
```

Comment:
This is not compatible with python 3, just use ``items()``. Yes, it's slightly inefficient in python 2, but you won't have a million options :-)